### ISSUE TITLE
iPad Air 2 (a1566) horrific constant buzzing from audio during play

### ISSUE THREAD

**Issue by RealRobertBrownieJr** [rileytestut/Delta#330]
Hey so yeah basically I have an iPad jailbroken using palera1n ios 15.7.2, installed delta (latest) copied over my roms etc

attempting to play them though results in a horrific loud buzzing is the only way I can describe it 
Like the gameplay itself seems to be ok visually and stuff but after launching any rom the game starts then comes the weird loud buzz.

**Comment by RealRobertBrownieJr** [comment]
If it helps I actually just recorded this while it happens 

https://github.com/rileytestut/Delta/assets/108153010/2f36603b-a48b-46a6-8889-105e17014f8e

**Comment by rmyj** [comment]
Is this the same with all of your games for GBA?

**Comment by dantezhoux** [comment]
> 你们所有的 GBA 游戏都是这样吗？

https://github.com/rileytestut/Delta/issues/330#issuecomment-2075756038

**Comment by dantezhoux** [comment]
> 你们所有的 GBA 游戏都是这样吗？

Yes, GBA and DS work normally for a short time, and then there is a strange buzzing sound. And, during normal operation, there may also be explosive sounds

**Comment by RealRobertBrownieJr** [comment]
> Is this the same with all of your games for GBA?

yeah same  thing seemingly for all gba and ds with ds lasting a few seconds longer but still doing the same thing

**Comment by GiantEnemyCrab** [comment]
The same thing happens with iPhone SE 1st gen 2016 with A9 processor.
